js/index.js

```diff
@@ -63,6 +63,28 @@ messageForm.addEventListener('submit', function(event) {
     messageForm.reset();
 });
 
+fetch('https://api.github.com/users/JGDel24/repos')
+    .then(function (response) {
+        if (response.ok) {
+            return response.json();
+        }
+    })
 
+    .then(function (repository) {
```

Comment:
Suggestion: Rename this variable to `repositories` to make it clear that it's collection of repositories, not a single one.